contents/apps/variance-connector/docs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Variance Connector documentation
+showTitle: true
+topics:
+    - variance-connector
+---
+
+### What does the Variance Connector app do?
+
+
+### What are the requirements for this app?
+
+
+### How do I install the Variance Connector app?
+
```

Comment:
```suggestion
To install the Variance app you'll need a Variance account. In your Variance account go to Variance > Integrations > Create a new PostHog connection. This will give you a Webhook URL and Authorization header value. You will then use those two values when installing the app in your PostHog instance. Don't forget to hit enable after you've added the configuration details.
```